Add limits on hours logged. Fixes #10

reeved opened the pull request:
Add a limit of 24 hours added/removed to stop larger numbers from being inputted

reeved committed:
Add limits on hours logged

Add a limit of 24 hours added/removed to stop larger numbers from being inputted

DarkMatterMatt reviewed (approved):
Looks good, should we use `Math.abs(hours)> 24` instead? Either way is fine

Also do we care if George spams our thing with `hours = 0`? Is it worth checking that `hours != 0`?

reeved commented:
Good point. How about: 

`if(Math.abs(hours)> 24) {}` and
`if(hours = 0){}` with a message like:

"Looks like you didn't contribute any volunteering time. Try again once you have!"

open to suggestions for a better message. I suck at being funny 😆

reeved committed:
Update limits on hours logged

-Prevents users from adding/removing more than 24 hours
-Prevents users from logging 0 hours
